docs/docs/pods.md

```diff
@@ -42,47 +41,53 @@ Currently, Marathon pods can only be created and administered via the `/v2/pods/
 1. Verify the status of your new pod:
 
     ```bash
-    http GET <ip>:<port>/v2/pods/simplepod::status
+    curl GET <ip>:<port>/v2/pods/simplepod::status
     ```
 
 1. Delete your pod:
 
     ```bash
-    http DELETE <ip>:<port>/v2/pods/simplepod
+    curl DELETE <ip>:<port>/v2/pods/simplepod
     ```
 
 # Technical Overview
 
-A pod is a special kind of [group](http://mesosphere.github.io/marathon/docs/application-groups.html), and the applications or containers in the pod are the group members.* A pod instance’s containers are launched together, atomically, via the Mesos LAUNCH_GROUP call. Containers in pods share networking namespace and ephemeral volumes.
+A pod is a special kind of Mesos task group, and the tasks or containers in the pod are the group members.* A pod instance’s containers are launched together, atomically, via the Mesos LAUNCH_GROUP call. Containers in pods share networking namespace and ephemeral volumes.
 
 \* Pods cannot be modified by the `/v2/groups/` endpoint, however. Pods are modified via the `/v2/pods/` endpoint.
 
 ## Networking
-Marathon pods only support the [Mesos containerizer](http://mesos.apache.org/documentation/latest/mesos-containerizer/), which simplifies networking among containers. The containers of each pod instance share a network namespace and can communicate over localhost. 
+Marathon pods only support the [Mesos containerizer](http://mesos.apache.org/documentation/latest/mesos-containerizer/). The Mesos containerizer supports multiple image formats, including Docker.
 
-Containers in pods are created with endpoints. Other applications communicate with pods by addressing those endpoints. If you specify a container network without a name in a pod definition, it will be assigned to this default network.
+The Mesos containerizer simplifies networking by allowing the containers of each pod instance to share a network namespace and communicate over localhost.
+
+If you specify a container network without a name in a pod definition, it will be assigned to this default network. <!-- check the pod def for more info on this -->
+
+If you neeed other applications to communicate with your pod, specify an endpoint in your pod definition. Other applications will communicate with your pod by addressing those endpoints. See [the Examples section](#endpoints) for more information.
 
 In your pod definition you can declare a `host` or `container` network type. Pods created with `host` type share the network namespace of the host. Pods created with `container` type use virtual networking. If you specify the `container` network type, you must also declare a virtual network name in the `name` field. See the [Examples](link) section for the full JSON.
```

Comment:
"you myst also declare a virtual network name in the `name` field"

.. maybe. it depends whether Marathon was started with a configured `--default_network_name`: if not, then `name` is required for `container` networks. Otherwise `name` is optional for `container` networks (as per the statement 2 paragraphs prior)